packages/react/src/navbar/navbar-expandable-menu.tsx

```diff
@@ -143,7 +143,7 @@ export const NavbarExpandableMenuContent = forwardRef<
       id={contentId}
       className={clsx("hds-navbar__expandable-menu-content", className as undefined)}
       data-state={open ? "open" : "closed"}
-      {...{ inert: open ? undefined : "true" }}
+      {...{ inert: open ? undefined : true }}
```

Comment:
This does no longer work as expected after this change. The links inside the expandable menu are now tab-able.